near-sdk/src/lib.rs

```diff
@@ -87,6 +87,8 @@
 //! ```bash
 //! cargo test
 //! ```
+//!
+//! # For more information, see the [macro@near] documentation.
```

Comment:
can you move it to line 27 and make a special emphasis it's cool and should be the main entry point for navigating the documentation,
, as suggested in https://github.com/near/near-sdk-rs/issues/1288#issuecomment-2564292206 ? 

nobody will notice it at the very end

Reply:
the rest of the boxes have probably already been killed with checkmarks

Reply:
maybe even duplicate it, if you absolutely want to have it at the very end